tests/robot-tests/tests/admin_and_public_2/bau/publish_release_and_amend.robot

```diff
@@ -576,13 +615,13 @@ Verify amended Dates data block table has footnotes
 
 Update second accordion section text for amendment
     user opens accordion section    Test text    id:releaseMainContent
-    user adds content to autosaving accordion section text block    Test text    1    Updated test text!
+    user adds content to autosaving accordion section text block    Test text    1    Amended test text!
     ...    id:releaseMainContent
 
 Update embedded dashboard title
     user updates embedded dashboard in editable accordion section
     ...    Test embedded dashboard section
-    ...    Test embedded dashboard title updated
+    ...    Test embedded dashboard title Amended
```

Comment:
Very minor thing.  In the same convention where you've added "Amended" to the *start* of an amended bit of text, I'd advise doing the same here:

`Amended Test embedded dashboard title`

and below on the check at line 811.

Just adds a bit of extra consistency to the test style ( I can see that you're just updating the text that was already in the test of course :) )